### PR TITLE
Feature/itinerary banner, fixed

### PR DESCRIPTION
**Pull request by gunnm0**
fixed errors that caused text in the tags to be cropped.

Screenshot from 2024-04-26 17-17-30

### CODE REVIEW

File: app/src/main/java/com/github/wanderwise_inc/app/ui/itinerary/ItineraryBanner.kt [hunk 1]
```diff
@@ -81,100 +83,96 @@ fun ItineraryBanner(
...
+              // Primary Text Field
+              Text(
+                  text = itinerary.title,
+                  color = MaterialTheme.colorScheme.onPrimaryContainer,
+                  fontFamily = FontFamily.Monospace,
+                  fontSize = 14.sp,
+                  fontWeight = FontWeight.Bold,
+                  modifier = Modifier.padding(2.dp)
+                  // textDecoration = TextDecoration.Underline
+                  )
```
Comment (ThomasPFMuller) [inline app/src/main/java/com/github/wanderwise_inc/app/ui/itinerary/ItineraryBanner.kt:95]: Good idea to put the Title on top

File: app/src/main/java/com/github/wanderwise_inc/app/ui/itinerary/ItineraryBanner.kt [hunk 2]
```diff
@@ -81,100 +83,96 @@ fun ItineraryBanner(
...
+                                Text(
+                                    text = it,
+                                    textAlign = TextAlign.Center,
+                                    color = MaterialTheme.colorScheme.onSecondaryContainer,
+                                    fontFamily = FontFamily.Monospace,
+                                    fontSize = 12.sp,
+                                    modifier = Modifier.padding(horizontal = 8.dp, vertical = 2.dp))
+                              }
+                        }
+                      }
```
Comment (ThomasPFMuller) [inline app/src/main/java/com/github/wanderwise_inc/app/ui/itinerary/ItineraryBanner.kt:149]: cleaner look, approved

**Review by ThomasPFMuller — APPROVED** [review]
Lgtm, cleaner look overall

**Review by 276746 — CHANGES_REQUESTED** [review]
Merge in dev

**Review by 276746 — APPROVED** [review]
All fine

**Pull request merged**